server/index.js

```diff
@@ -1,12 +1,68 @@
 const express = require('express');
+const sampleData = require('../database/sampleDatas/imageServiceSample.js')
 
 
 const app = express();
 
 app.use(express.static('public'));
+app.use(express.json());
+app.use(express.urlencoded())
 
 //console.log(__dirname + '/public/lib')
 
-var port = 3000;
+app.get('/:productId', (req, res) => {
+  var product_id = parseInt(req.params.productId);
+  var data = sampleData.find(data => data.product_id === product_id);
+  var type = req.query.type;
+
+  if (!type && product_id === data.product_id) {
+    res.send(data);
+  }
+
+  if (type === 'carousel_images') {
+    res.send({
+      product_id: data.product_id,
+      carousel_images: data.carousel_images

```

Comment:
If you do the destructuring thing above you can simplify this a lot. If the key and value are the same you only need to put it once. ie if you destructure out `product_id` then you could have `{ product_id: data.product_id }` which can be further simplified to just be `{ product_id }`.